lib/src/chewie_player.dart

```diff
@@ -277,7 +277,7 @@ class ChewieController extends ChangeNotifier {
   final List<OptionItem> Function(BuildContext context)? additionalOptions;
 
   /// Define here your own Widget on how your n'th subtitle will look like
-  final Widget Function(BuildContext context, String subtitle)? subtitleBuilder;
+  final Widget Function(BuildContext context, dynamic subtitle)? subtitleBuilder;
```

Comment:
`dynamic` is dangerous and should always be avoided (see Flutter Code-Guidelines). If we want to provide a custom `text` and it belongs to the UI, it should be a Widget here, so the User can decide what kind of Widget she/he wants to add here.

Reply:
@Ahmadre using a `Widget` inside of `Subtitle` data object is probably not a good idea, i.e. to keep UI rendering separate from the values.

How about using generic value in `Subtitle<T>` and `Subtitles<T>` classes?

Then `T` could be `String`, `InlineSpan`. That may require to move the `subtitleBuilder` function into `Subtitles` to tie it in to the same generic type. Also this approach won't allow a mix of different text types like in my original PR.